sqlglot/parser.py

```diff
@@ -1035,7 +1035,7 @@ def _parse_update(self):
         return self.expression(
             exp.Update,
             **{
-                "this": self._parse_table(schema=True),
+                "this": self._parse_table(alias_tokens=self.TABLE_ALIAS_TOKENS - {TokenType.SET}),
```

Comment:
make this a constant under TABLE_ALIAS_TOKENS where it is defined

Reply:
ohhhh ok, guess it makes sense to have constants in the same place instead of doing it on the fly